setup.py

```diff
@@ -256,7 +257,9 @@ def run(self):
 
 extras["serving"] = deps_list("pydantic", "uvicorn", "fastapi", "starlette")
 extras["audio"] = deps_list("librosa", "pyctcdecode", "phonemizer")
-extras["speech"] = deps_list("torchaudio") + extras["audio"]  # `pip install ".[speech]"` is deprecated and `pip install ".[torch-speech]"` should be used instead
+extras["speech"] = (
+    deps_list("torchaudio") + extras["audio"]
+)  # `pip install ".[speech]"` is deprecated and `pip install ".[torch-speech]"` should be used instead
```

Comment:
Put the comment above to avoid the ugly restyling like this.

Reply:
Done in `tokenizers` PR (and rebased since)